### PR TITLE
Migrate tests using `evaluate_common` to null-safety

### PR DESCRIPTION
**Pull request by elliette**

### CODE REVIEW

File: dwds/test/evaluate_common.dart [hunk 1]
```diff
@@ -101,19 +99,19 @@ void testAll({
...
+      late ScriptRef testLibraryScript;
+      late ScriptRef testLibraryPartScript;
+      late Stream<Event> stream;
 
       setUp(() async {
         setCurrentLogWriter(debug: debug);
         vm = await setup.service.getVM();
-        isolate = await setup.service.getIsolate(vm.isolates.first.id);
-        scripts = await setup.service.getScripts(isolate.id);
+        isolate = await setup.service.getIsolate(vm.isolates!.first.id!);
```
Comment (annagrin) [inline dwds/test/evaluate_common.dart:113]: add a non-nulable isolateId variable for convenience?

File: dwds/test/evaluate_common.dart [hunk 2]
```diff
@@ -507,41 +505,41 @@ void testAll({
...
+                  .evaluateInFrame('bad', event.topFrame!.index!, 'local'),
               throwsSentinelException);
         });
       });
     });
 
     group('evaluate', () {
       VM vm;
-      Isolate isolate;
+      late Isolate isolate;
```
Comment (annagrin) [inline dwds/test/evaluate_common.dart:522]: add a non-nullable `isolateId` variable for convenience?

File: dwds/test/evaluate_common.dart [hunk 3]
```diff
@@ -605,35 +603,35 @@ void testAll({
 
     group('evaluateInFrame', () {
       VM vm;
-      Isolate isolate;
+      late Isolate isolate;
```
Comment (annagrin) [inline dwds/test/evaluate_common.dart:606]: similar here

**Review by annagrin — APPROVED** [review]
LGTM with some comments.

File: dwds/test/evaluate_common.dart (hunk 1 above)
Comment (elliette) [reply dwds/test/evaluate_common.dart:114]: Done

File: dwds/test/evaluate_common.dart (hunk 3 above)
Comment (elliette) [reply dwds/test/evaluate_common.dart:611]: Done

File: dwds/test/evaluate_common.dart (hunk 2 above)
Comment (elliette) [reply dwds/test/evaluate_common.dart:525]: done

**Pull request merged**